vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -1893,6 +1904,10 @@ func expandClusterDrsConfigInfo(d *schema.ResourceData) *types.ClusterDrsConfigI
 		Option:                    expandResourceVSphereComputeClusterDrsAdvancedOptions(d),
 	}
 
+	if version.Newer(viapi.VSphereVersion{Product: version.Product, Major: 7, Minor: 0}) {
```

Comment:
I need to check the precedent of this throughout the provider, but would it be appropriate to error if they tried configuring this field on < V7 of vSphere VS noop silently?

Reply:
Ah realizing now this is an expansion function... makes error propagation pretty awkward.

Reply:
Does this one need a change?

Reply:
Seems like you are following precedant... Once vsphere 6 is EOL and we rip all the checks out there should be a new precedent for version validations. All good as is.